frontend/src/components/header/Header.js

```diff
@@ -1,10 +1,17 @@
+import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
 
-export default function Header() {
+export default function Header({ className, text, secondary }) {
+  Header.propTypes = {
+    className: PropTypes.string,
+    text: PropTypes.string.isRequired,
+    secondary: PropTypes.bool,
+  }
+
   return (
-    <HeaderStyled>
-      <HeadingContainer>
-        <Heading>My Plants</Heading>
+    <HeaderStyled className={className}>
+      <HeadingContainer secondary={secondary}>
+        <Heading secondary={secondary}>{text}</Heading>
```

Comment:
Secondary is a boolean? Why don't call it something like isSecondary?

Reply:
true